Reference/Routing/custom-controllers.md

```diff
@@ -2,167 +2,246 @@
 versionFrom: 8.0.0
 ---
 
-# Custom controllers (Hijacking Umbraco Routes)
+# Custom MVC controllers (Umbraco Route Hijacking)
 
-_If you need complete control over how your pages are rendered then Hijacking Umbraco Routes is for you_
+_Use a custom MVC controller to handle and control incoming requests for content pages based on a specific Document Type_
 
-## What is Hijacking Umbraco Routes?
+## What is Umbraco Route Hijacking?
 
-By default all of the front end routing is executed via the `Umbraco.Web.Mvc.RenderMvcController` Index Action which should work fine for most people. However, in some cases people may want complete control over this execution and may  want their own Action to execute. Some reasons for this may be: to control exactly how views are rendered, custom/granular security for certain pages/templates or to be able to execute any custom code in the controller that renders the front end. The good news is that this is completely possible.
+By default, all front end requests to an Umbraco site are auto-routed via the 'Index' action of a core MVC Controller: `Umbraco.Web.Mvc.RenderMvcController`. This core controller handles the incoming request, builds the associated PublishedContent view model, and passes this to the appropriate Umbraco Template/MVC View. 
+
+It is however possible to implement a custom MVC Controller to replace this default implementation to give complete control over this execution.
+
+For example, to enrich the view model used for the template with additional properties perhaps retrieved from other published content items or even outside of Umbraco, or to implement serverside paging, or any custom/granular security or business logic required to actioned before rendering the template/view on the front-end.
+
+This replacement of the default controller can either be achieved 'globally' for all requests or by 'hijacking' requests for pages based on a specific Document Type following a simple controller naming convention [doctypealias]Controller. 
 
 ## Creating a custom controller
 
- Let's demonstrate with an example : let's say you have a Document Type called 'Home'.  You can create a custom locally declared controller in your MVC web project called 'HomeController' and ensure that it inherits from `Umbraco.Web.Mvc.RenderMvcController`. Now all pages that are of document type 'Home' will be routed through your custom controller!
+### Example: hijacking route requests to a 'product' page
+
+If the Umbraco site has 'product' pages created from a Document Type called 'Product Page' with alias 'productPage'.
 
-In order for you to run some code in your controller you'll need to override the Index Action. Here’s an example:
+Then, creating a custom locally declared controller in the Umbraco MVC web application project (or App_Code folder for website projects) named 'ProductPageController'.
 
+And, ensuring this controller inherits from the base `Umbraco.Web.Mvc.RenderMvcController`.
```

Comment:
```suggestion
And, we will ensure that this controller inherits from the base `Umbraco.Web.Mvc.RenderMvcController`.
```

Reply:
Again it's not meant to be a tutorial.. maybe it should be :-P

Reply:
Ok think I have resolved! have taken out the extra words!